assets/dev/js/editor/elements/views/behaviors/context-menu.js

```diff
@@ -55,6 +55,27 @@ module.exports = Marionette.Behavior.extend( {
 			} );
 		}
 
+		// Adding custom groups will only be possible for elements' context menus (Section, Column, Widget).
+		if ( this.view.options.model?.get( 'elType' ) ) {
```

Comment:
@udidol Try to find a way to do that without having to do this explicit check here